spec/fixtures/files/wpscan_upload_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+
+describe 'wpscan upload plugin' do
+  describe "Importer" do
+
+    before(:each) do
+      # Stub template service
+      templates_dir = File.expand_path('../../templates', __FILE__)
+      expect_any_instance_of(Dradis::Plugins::TemplateService)
+      .to receive(:default_templates_dir).and_return(templates_dir)
+
+      # Init services
+      plugin = Dradis::Plugins::Wpscan
+
+      @content_service = Dradis::Plugins::ContentService::Base.new(plugin: plugin)
+
+      @importer = plugin::Importer.new(
+        content_service: @content_service,
+      )
+    end
+
+    it "raises an error note when the json is not valid" do
+      # Run the import
+      p @importer.import(file: 'spec/fixtures/files/invalid.json')
+    end
+
+    # it "creates nodes, issues, notes and an evidences as needed" do
```

Comment:
@ethicalhack3r this should be uncommented, yes?

Reply:
Yea, was just debugging at the time and didn't want to lose my local changes.

I had a quick look earlier but came across another couple of issues. I will have another look tomorrow, and take a look at some other plugin spec examples, and hopefully get it finished.